base/src/Data/Macaw/Memory/ElfLoader.hs

```diff
@@ -690,6 +690,18 @@ relaTargetARM32 end msegIndex symtab rel addend relFlag =
                          , relocationEndianness = end
                          , relocationJumpSlot   = True
                          }
+    Elf.R_ARM_COPY -> do
+      sym <- resolveRelocationSym symtab (Elf.relSym rel)
+      when (addend /= 0) $ do
+        throwError $ RelocationUnsupportedType (show (Elf.relType rel))
+      pure $ Relocation { relocationSym        = sym
+                        , relocationOffset     = 0
+                        , relocationIsRel      = False
+                        , relocationSize       = 4
+                        , relocationIsSigned   = False
+                        , relocationEndianness = end
+                        , relocationJumpSlot   = False
+                        }
```

Comment:
This code is almost entirely copy-pasted from the `R_X86_64_COPY` case above, except that I changed the `relocationSize` from `8` to `4`. I believe the other fields don't change for AArch32.